docs/InstallationConfiguration.md

```diff
@@ -17,133 +17,6 @@ Several files need to be created/updated for LEAF to operate in your environment
 
 In the sections below `$dbUser` and `$dbPass` are the same values used in the mysql Dockerfile and setup script.
 
-## LEAF_Nexus
```

Comment:
When setting  this up from scratch does this setup the sites table now? Also sites table may be a good note to have here  in any case as a troubleshooting step. Will help people who may not be as familiar with the project.